### PR TITLE
feat: Material UI v5 migrate

### PR DESCRIPTION
**Pull request by ChrisBaidoo**
## Description

Migration to MaterialUI v5

Have any database changes been committed as part of this pull request?

## Checklist:

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request). If you're unsure about any of these, don't hesitate to leave a comment on this pull request!

- [ ] I have read the gliff.ai Contribution Guide.
- [ ] I have requested to **pull a branch** and not from main.
- [x] I have checked all commit message styles match the requested structure.
- [x] My code follows the style guidelines of this project.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have performed a self-review of my own code.
- [x] I have assigned **3 or less** reviewers.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] My changes generate no new warnings.
- [ ] I have made corresponding changes to the documentation.
- [ ] New database changes have been committed.
- [x] If appropriate, I have bumped any version numbers.

### CODE REVIEW

**Review by joshuajames-smith — APPROVED** [review]
UI/UX side, staging seems all good! (minus Percy not taking a liking to it)

File: package.json [hunk 1]
```diff
@@ -58,13 +55,16 @@
...
   },
   "peerDependencies": {
-    "@material-ui/core": "^4.11.3",
-    "@material-ui/icons": "^4.11.2",
-    "@material-ui/lab": "^4.0.0-alpha.58",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
   },
   "dependencies": {
+    "@emotion/react": "^11.7.1",
```
Comment (ChasNelson1990) [inline package.json:62]: Again, these should stay as peer and dev Deps, rather than be moved into deps.

**Review by ChasNelson1990 — APPROVED** [review]

**Comment by cooper667** [comment]
👍  to the libs being peer/dev dependencies.

This should also be a major version bump as it's incredibly breaking?

**Comment by ChasNelson1990** [comment]
Good catch!

**Comment by cooper667** [comment]
I think we'll have to mark 3.1.0 as deprecated to stop it getting dependaboted

**Pull request merged**